styles/stylesheet.css

```diff
@@ -0,0 +1,437 @@
+   @media print {
+     body img {
+        display: none;
+     } 
+   }
+   @media screen and (min-width: 200px) {
+       .homeimage {
+          height: 400px;
+          object-fit: contain;
+       }
+   }
+   @media screen and (min-width: 200px) {
+       .blogimage2 {
+          height: 200px;
+          object-fit: contain;
+          padding-top: 30px;
+       }
+   }
+   @media screen and (min-width: 200px) {
+       img {
+         display: inline;
+          width: 100%;
+       }
+   }
+   @media screen and (max-width: 768px) {
+        .hidetext {
+          display: none;
+       }
+   }
+
+   @media screen and (max-width: 768px) {
+        .box-content1 {
```

Comment:
On this one, we're looking for you to manually set the grid columns within the media queries, as opposed to using auto and setting a manual width.